debug-salt

```diff
@@ -0,0 +1,751 @@
+#!/usr/bin/perl -w
```

Comment:
Add a copyright header - something like this matches the other piece of this plugin:

	#############################################################
	# Name:        Salt Events Parser for SUSE CaaS Platform
	# License:     GPLv2
	#############################################################

Reply:
I have added a copyright header at the top. Have amended the current commit.